paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -1480,7 +1529,7 @@ def check_if_instance_is_done(
     service: str,
     instance: str,
     cluster: str,
-    git_sha: str,
+    version: DeploymentVersion,
```

Comment:
Sometimes we're passing in `git_sha` and `image_version` separately, and other times together as `DeploymentVersion`, do we have a preference for one over the other in general?

Reply:
It's a good question; one I asked myself multiple times.  It might help keep things neater to prefer a DeploymentVersion and make sure it helps future devs conceptualize what a 'version' is better, without having to remember the set of params that make up that information.

In truth I had just been changing to deploymentversion in places where I didn't want to add `image_version` as a kwarg with default it to `None` since we have so many functions w/ only positional args (but later realized we just needed to make sure we passed `None` from its callers....)